lib/matplotlib/axes/_base.py

```diff
@@ -3283,8 +3284,11 @@ def ticklabel_format(self, *, axis='both', style='', scilimits=None,
             except (ValueError, TypeError) as err:
                 raise ValueError("scilimits must be a sequence of 2 integers"
                                  ) from err
-        STYLES = {'sci': True, 'scientific': True, 'plain': False, '': None}
-        is_sci_style = _api.check_getitem(STYLES, style=style)
+        STYLES = {'sci': True, 'scientific': True, 'plain': False}
```

Comment:
An option here (and also in case someone has set `style=''` explicitly, is to do:
```suggestion
        STYLES = {'sci': True, 'scientific': True, 'plain': False, '': None, None: None}
        is_sci_style = _api.check_getitem(STYLES, style=style)
```
and just keep the single line

Reply:
> in case someone has set `style=''` explicitly

That's a good point.  I suggest to also add a comment that `''` is included for backwards-compatibility, as I do not think it would be obvious to future readers of the code.